RFPStrategy/RFP/spec.ts

```diff
@@ -0,0 +1,44 @@
+import { Address, BeforeAll, BigInt, Event, LiveObject, OnEvent, Property, Spec } from '@spec.dev/core'
+
+/**
+ * RFP details
+ */
+@Spec({
+    uniqueBy: ['strategyId', 'chainId']
+})
+class RFP extends LiveObject {
+
+    @Property()
+    strategyId: Address
+
+    @Property()
+    active: boolean
+
+    @Property({ default: 0 })
+    maxBid: BigInt
+
+    // ====================
+    // =  Event Handlers  =
+    // ====================
+
+    @BeforeAll()
+    setCommonProperties(event: Event) {
+        this.strategyId = event.origin.contractAddress;
+    }
+
+    // TODO-SPEC: Is this needed cause RFPCommitteeStrategy extends RFPSimple 
```

Comment:
@whittlbc

Reply:
Yeah I would still include these. Otherwise you'll never receive the `PoolActive` or `MaxBidIncreased` events from any strategy contracts in the `allov2.RFPCommitteeStrategy` group.